### PR TITLE
Get animation group by node

### LINKED ISSUE #228: [Max][AnimationGroups] Find AnimationGroup(s) that use(s) a node
Artists requested to have a way to select the animation groups in which a max node is used. Because we have a lot of animations (>100 in some cases), the artists could really use this to fix things (e.g. to figure out if a mesh is used in the wrong animation).

### PR DESCRIPTION
**Pull request by elpie89**
Fixes #228

### CODE REVIEW

**Comment by deltakosh** [comment]
Thanks a lot for your appreciated help!!

pinging @Drigax

File: 3ds Max/Max2Babylon/Forms/AnimationForm.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ private void AnimationForm_Load(object sender, EventArgs e)
         private void createAnimationButton_Click(object sender, EventArgs e)
         {
             AnimationGroup info = new AnimationGroup();
-            
+
```
Comment (Drigax) [inline 3ds Max/Max2Babylon/Forms/AnimationForm.cs]: can we please minimize white space changes not related to code changes. it makes git-blame associate you with stuff you may not have changed substantially.

I always appreciate code cleanup, but I'd like to avoid hygienic changes that aren't part of a behavioral change.

File: 3ds Max/Max2Babylon/Forms/AnimationForm.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ private void deleteAnimationButton_Click(object sender, EventArgs e)
                 return;
 
             AnimationGroup selectedItem = (AnimationGroup)AnimationListBox.SelectedItem;
```
Comment (Drigax) [inline 3ds Max/Max2Babylon/Forms/AnimationForm.cs]: same here, we can do this in a separate PR

**Comment by deltakosh** [comment]
Up @elpie89

**Comment by elpie89** [comment]
Should be fixed now, this white space stuff is getting me crazy,
I'm trying to understand how to configure vs to avoid this.
Do we have any guidelines for developers who join the project?

**Comment by Drigax** [comment]
Best practice is to usually look over your changeset before commit via `git diff`. I'm assuming you're developing on windows as well, best thing to do in this case is to configure git to checkout windows-style, but commit unix-style line endings, so that git doesn't mangle them for a whole file on checkin.

I usually enable visual whitespace in VS, so that I can see spaces as `.`

Also, it never hurts to look over your pull request after creation

**Comment by elpie89** [comment]
Yes, you are right, I'm pretty new to open source project, and I always worked on perforce.
I'll try to avoid these kinds of problems in the future.

**Comment by Drigax** [comment]
Thanks for the contribution! changes look good!

**Comment by Drigax** [comment]
I broke the project when resolving the merge conflicts from this commit. I'll have to retry merging this manually.

**Comment by elpie89** [comment]
@Drigax 
this merge broke something, multiselection, clean and merge are no longer there
Let me know if I can help from my side

**Comment by Drigax** [comment]
hey @elpie89 yeah, I had to revert the original merge commit. I re-added them via #527. can you check that this properly re-implemented them before I publish the next build number?
